server/controllers/teacher/delete-quiz.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Response } from 'express';
+import { deleteQuiz } from '../../database/queries';
+import { CustomError } from '../../errors';
+import { UserAuth } from '../../interfaces';
+
+export default async (req:UserAuth, res:Response, next:NextFunction) => {
+  const { params } = req;
+
+  try {
+    const { rowCount } = await deleteQuiz(params.quizId);
```

Comment:
You already destructured the params, complete your favor and destructure the quizId and then validate it